treeherder/config/settings.py

```diff
@@ -18,20 +20,38 @@
 
 env = environ.Env()
 
+# Checking for OS type
+IS_WINDOWS = "windows" in platform.system().lower()
+
 # Top Level configuration
 DEBUG = env.bool("TREEHERDER_DEBUG", default=False)
 
+#  Default environment variables check
+REQUIRED_PRODUCTION_ENV_VARIABLES = ["BROKER_URL",
+                                     "DATABASE_URL",
+                                     "NEW_RELIC_DEVELOPER_MODE",
```

Comment:
My apologies I missed this one. This needs removing.

Reply:
Sorry sir. Are you referring to the comment?

Reply:
Understood. Last line of the highlighted code is the line intended on Github!